lib/mobility/plugins/active_record/original_column.rb

```diff
@@ -0,0 +1,60 @@
+# frozen-string-literal: true
+
+module Mobility
+  module Plugins
+=begin
+
+Plugin to use an original column for a given locale, and otherwise use the backend.
+
+=end
+    module ActiveRecord
+      module OriginalColumn
+        extend Plugin
+
+        requires :original_column, include: false
+
+        included_hook do |_, backend_class|
+          if options[:original_column]
+            backend_class.include BackendInstanceMethods
+            backend_class.extend BackendClassMethods
+          end
+        end
+
+        module BackendInstanceMethods
+          def read(locale, **)
+            if locale == I18n.default_locale
```

Comment:
I don't love this dependence on `I18n` ... :thinking:

Reply:
I actually only used `I18n.default_locale` as an example. In practice, I actually need it to work with `en` OR `en-US`, where `en-US` falls back to plain `en`.

So this check would for sure be configurable -- either a static plugin config, or maybe a proc called in-context when building a new instance.

Reply:
It could be optionally a proc which would be evaluated in context. The problem is that then we need to use `define_method` instead of `def` which has (small) perf implications, which I try to avoid on anything that runs through attribute getters/setters.